README.md

```diff
@@ -15,22 +15,26 @@ implementation, such as [up-transport-zenoh-cpp][zenoh-transport-repo].
 
 ### Requirements:
 - Compiler: GCC/G++ 11 or Clang 13
-- Conan : 1.59 or latest 2.X
+- pip : To install others
+- Conan(pip) : 1.59 or > 2.4.2
+- Cmake(apt) : > 3.22
+- Ninja(pip) : Latest
+- Gcovr(pip) : For UT code coverage if desired
```

Comment:
Do not specify the package manager for installing tools. That will be different for every user, build environment, and distro. Developers are expected to know how to source and install tools for their environment.

Reply:
Resolved.  Actually, three of those are platform independent (all installed w/ pip).